admin/partials/settings/compressor.php

```diff
@@ -53,6 +53,29 @@
 }
 $select_compressor .= '</select>';
 
+// Create the Compression level options.
+$default_compression_level  = 6;
+$selected_compression_level = isset( $settings['compression_level'] ) ? $settings['compression_level'] : $default_compression_level;
+$select_compression_level   = '<select name="compression_level">';
+foreach ( range( 0, 9 ) as $level ) {
+	$is_default  = $default_compression_level === $level;
+	$is_selected = (int) $selected_compression_level === $level;
```

Comment:
Why run `(int)` during every iteration of the loop? Can we just do it once above?